deployment/src/test/java/io/quarkiverse/loggingui/quarkus/logging/ui/deployment/AbstractConfigurationTest.java

```diff
@@ -0,0 +1,130 @@
+        String loggerName = "ROOT";
+        Map<String, Object> response = performGetRequest(path, loggerName);
+        checkGetResponse(response, loggerName);
+    }
+
+    void postEndpointSetsLogLevel(String path) {
+        String loggerName = "ROOT";
+        String newLogLevel = "TRACE";
+        Map<String, Object> response = performGetRequest(path, loggerName);
+        checkGetResponse(response, loggerName);
+        performPostRequest(path, loggerName, newLogLevel, 201);
+        Map<String, Object> secondCallResponse = performGetRequest(path, loggerName);
+        checkGetResponse(secondCallResponse, loggerName, newLogLevel, newLogLevel);
+        // Put it back to where is was
+        performPostRequest(path, loggerName, "INFO", 201);
+
+    }
+
+    void postEndpointSetsLogLevelAlsoInLowerCase(String path) {
+        String loggerName = "io.quarkus.loggers";
+        String newLogLevel = "trace";
+        Map<String, Object> response = performGetRequest(path, loggerName);
+        checkGetResponse(response, loggerName);
+        performPostRequest(path, loggerName, newLogLevel, 201);
+        Map<String, Object> secondCallResponse = performGetRequest(path, loggerName);
+        checkGetResponse(secondCallResponse, loggerName, newLogLevel.toUpperCase(Locale.ROOT),
+                newLogLevel.toUpperCase(Locale.ROOT));
+        // Put it back to where is was
+        performPostRequest(path, loggerName, "INFO", 201);
+    }
+
+    void postEndpointFailsOnUnknownLogLevel(String path) {
+        String loggerName = "ROOT";
+        String newLogLevel = "non-existing";
+        performPostRequest(path, loggerName, newLogLevel, 400);
+    }
+
+    private Map<String, Object> performGetRequest(String path, String loggerName) {
+
+        return RestAssured.when().get(path + "?loggerName=" + loggerName).then().statusCode(200)
+                .body(notNullValue())
+                .extract()
+                .as(Map.class);
+    }
+
+    private void checkGetResponse(Map<String, Object> response, String loggerName) {
+        checkGetResponse(response, loggerName, null, "INFO");
+    }
+
+    private void checkGetResponse(Map<String, Object> response, String loggerName, String configuredLogLevel,
+            String effectiveLogLevel) {
+
+        assertThat(response.get("name"), equalTo(loggerName));
+        assertThat(response.get("effectiveLevel"), equalTo(effectiveLogLevel));
+    }
+
+    private void performPostRequest(String path, String loggerName, String newLogLevel, int expectedStatusCode) {
+        given().contentType("application/json")
+                .body(String.format("{\n" +
+                        "	\"name\": \"%s\",\n" +
+                        "	\"configuredLevel\": \"%s\"\n" +
+                        "}", loggerName, newLogLevel))
+                .when()
+                .post(path)
+                .then().statusCode(expectedStatusCode);
+    }
+
+    // TODO:
+    // Other technologies (slf4j?) compatible?
```

Comment:
Would you have an idea on how to do that? Or an opinion whether this is a real requirement?

Reply:
I was thinking that we can use JBoss Logger (as that is already there) and JBoss Logger is an facade onto other logger.